lib/oli_web/live/community_live/table_model.ex

```diff
@@ -27,15 +28,20 @@ defmodule OliWeb.CommunityLive.TableModel do
           name: :actions,
           label: "Actions",
           render_fn: &__MODULE__.render_overview_button/3
+        },
+        %ColumnSpec{
+          name: :status,
+          label: "Status",
+          render_fn: &__MODULE__.render_status/3
         }
       ],
       event_suffix: "",
       id_field: [:id]
     )
   end
 
-  def render_overview_button(assigns, community, _) do
-    route_path = Routes.live_path(OliWeb.Endpoint, OliWeb.CommunityLive.Show, community.id)
+  def render_overview_button(assigns, %Community{id: community_id}, _) do
```

Comment:
Ok to just use `id` here - its clear that its a community id

Reply:
We should disable the `overview` button if the community is `deleted`, right?

Reply:
@mcass19 we're redirecting to the index page and showing a flash message accordingly when attempting to see the details of a deleted community (followed the same behavior as projects and products since that was the requirement).
It's a quick change so I'll make it disabled for `deleted` communities 👍